### DISCUSSION TITLE
Git 컨벤션 추가: PR 규칙

### DISCUSSION THREAD

**Discussion by zeus6768** [woowacourse-teams/2024-code-zap#144] [규칙]
### 1. RC를 구두로 해서 작업하는사람이 피곤함
- 피곤함
- 기록에 남지 않음

### 2. PR 마감기한 없음
- 내일 데모데이인데 아직도 업데이트 하고있음
- 우선순위 파악 어려움

### 3. comment 반영 중요도 파악 어려움
- 반영 할 필요 없다고 생각해서 안 했는데, 또 물어봄

**Comment by zeus6768** [comment]
## 해결

### 1
- RC는 Comment로 해결하는 것을 원칙으로 한다. 
- Comment로 해결되기 어려울 경우에만 구두로 의견을 나눈다. 

### 2
- PR에 zap 태그와 함께 마감시간을 명시한다. 

### 3
- 의견 반영 요청은 ⚡ 사용
- 의견 반영 완료는 🚀 사용
- 리뷰어가 resolve 한다.

**Comment by kyum-q** [comment]
## 추가 규칙
- 리뷰 중⚡ 이 아닐 경우, 리뷰이가 받아들이지 않아도 암말하지마 !
- ⚡ 인 경우에만 더 이야기 가능 ~
